doc/DEVELOPERS.rst

```diff
@@ -121,26 +106,20 @@ caused by these changes in underlying dependencies.
 .. __: https://mupdf.com/
 .. __: https://www.reportlab.com/
 
-To run all the tests enable your venv first if it's not enabled and then call ``pytest``::
+To run all the tests via ``pytest`` use::
 
-    pytest
+    uv run pytest
 
 You can also run tests in parallel using ``pytest-xdist`` by passing the ``-n auto`` flag.
 
-Firstly install::
-
-    pip install pytest-xdist
```

Comment:
Moved `pytest-xdist` into the dev dependencies defined in `pyproject.toml` rather than just a suggestion in the docs.

Reply:
Good idea. It's so much faster when testing locally :)

Reply:
I love both this change and the commentary that helped me understand it!